shoop/admin/modules/products/views/package/__init__.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2016, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+
+from .edit_package import ProductPackageView  # noqa
```

Comment:
Each object should be visible only in one place. Decide if it should be
* (a) `shoop.admin.modules.products.views.package.edit_package.ProductPackageView`, or
* (b) `shoop.admin.modules.products.views.package.ProductPackageView`, or maybe even
* (c) `shoop.admin.modules.products.views.ProductPackageView`

Then make it visible only there. To do that, either don't import it to other places (case (a)) or in case of (b) or (c) make the module private `_edit_package.py` and make it "visible" by importing it to somewhere and adding it to `__all__`.

There should be no need for `noqa` tags for this.